docs/0-zarf-overview.md

```diff
@@ -11,13 +11,13 @@ import Tabs from "@theme/Tabs";
 
 ## What is Zarf?
 
-Zarf was created to _**support the declarative creation & distribution of software "packages" into remote/constrained/independent environments**_.
+Zarf was created to _**support the declarative creation & distribution of software "packages" into remote/constrained/standalone environments**_.
 
 > "Zarf is a tool to help deploy modern stacks into air-gapped environments; it's all about moving the bits." &mdash; Jeff
 
-Zarf is a free and open-source tool that simplifies the setup and deployment of applications and resources onto air-gapped or disconnected environments. Zarf equips you with the ability to quickly and securely deploy modern software onto complex systems without relying on internet connectivity.
+Zarf is a free and open-source tool that simplifies the setup and deployment of applications and supporting resources onto air-gapped or disconnected environments. Zarf equips you with the ability to quickly and securely deploy modern software onto complex systems without relying on internet connectivity.
 
-It also simplifies the installation, updating, and maintenance of DevSecOps capabilities like Kubernetes clusters, logging, and SBOM compliance out of the box. Most importantly, Zarf keeps applications and systems running even when they are disconnected.
+It also simplifies the installation, update, and maintenance of supporting DevSecOps capabilities like Kubernetes clusters and logging stacks, while  providing features like out of the box SBOM compliance. Most importantly though, Zarf keeps applications and systems running even when they are disconnected.
```

Comment:
Reccomendation -  leaning into what we hoped to educate them on and add
"while providing features that ensure supply chain security, including SBOM compliance"

Reply:
done